plugins/modules/docker_compose_v2.py

```diff
@@ -0,0 +1,923 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright Léo El Amri
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: docker_compose_v2
+
+short_description: Manage multi-container Docker applications with Docker Compose.
+
+description:
+  - Uses Docker Compose to start and shutdown services.
+  - Swarm mode is not supported (thus secrets and configs are not supported).
+  - Configuration can be read from a Compose file or inline using the I(definition) option.
+  - See the examples for more details.
+
```

Comment:
```suggestion

extends_documentation_fragment:
  - community.docker.attributes

```

Reply:
I believe the docker_compose_v2 module does not support what's documented in this fragment.

Reply:
That's fine, as long as it documents that it doesn't support them with `support: none` :) You really need to add this documentation fragment, otherwise the sanity tests will continue failing.